sdk/nodejs/package.json

```diff
@@ -71,6 +71,7 @@
         ]
     },
     "//": [
-        "NOTE: @types/node is pinned due to grpc/grpc-node#2002"
+        "NOTE: @types/node is pinned due to grpc/grpc-node#2002",
+        "NOTE: @grpc/grpc-js is pinned due to grpc/grpc-node#2604"
```

Comment:
Issue thread says that 1.9.9 fixes this. Should we just try setting that as a new minimum?

Reply:
https://github.com/pulumi/pulumi/pull/14469

Sounds good.

Reply:
Bumped